src/server/svr_recov_db.c

```diff
@@ -477,6 +477,15 @@ sched_save_db(pbs_sched *ps, int mode)
 	obj.pbs_db_obj_type = PBS_DB_SCHED;
 	obj.pbs_db_un.pbs_db_sched = &dbsched;
 
+	/*
+	 * default values for scheduler object
+	 */
+	if ((ps->sch_attr[(int)SCHED_ATR_sched_cycle_len].at_flags & ATR_VFLAG_SET) == 0) {
```

Comment:
why are you setting an attribute while saving the object in db. Shouldn't this be done as part of sched_alloc itself?